README.md

```diff
@@ -7,6 +7,19 @@ available on Uppmax, and the individual READMEs may also be incomplete in terms
 of what was actually done to install the modules.  We are publicising these in
 the hopes that they can be helpful.
 
+Example workflow of a basic installation
+-------
+1. Clone the install methods git repo (`git clone https://github.com/UPPMAX/install-methods.git`)
+1. Add the repo to your `$PATH` and source the `uppmax_functions.sh` file to get access to the functions.
+1. Run `run_makeroom` with at least `-t` and `-v`, to generate a `.sh` (`makeroom_toolname_version.sh`) file that will create the directory structure needed in `/sw`
```

Comment:
Ok, we used run_makeroom since that is what you showed us. Looking at the code, run_makeroom removes the temporary variable file for you which seems better, keeping down the litter. Leaving it as is. 
Remember, this is just a quick-start guide for newbies, not documentation of the tools.